plugin-gradle/src/main/java/com/diffplug/gradle/spotless/KotlinExtension.java

```diff
@@ -90,20 +90,37 @@ public KtfmtConfig ktfmt() {
 	 * code.
 	 */
 	public KtfmtConfig ktfmt(String version) {
+		return ktfmt(version, false);
+	}
+
+	/**
+	 * Uses the given version of [ktfmt](https://github.com/facebookincubator/ktfmt) and applies the dropbox style
+	 * option to format source code.
+	 */
+	public KtfmtConfig ktfmt(String version, Boolean withDropboxStyle) {
 		Objects.requireNonNull(version);
-		return new KtfmtConfig(version);
+		Objects.requireNonNull(withDropboxStyle);
+		return new KtfmtConfig(version, withDropboxStyle);
 	}
 
 	public class KtfmtConfig {
 		final String version;
+		final Boolean withDropboxStyle;
 
 		KtfmtConfig(String version) {
 			this.version = Objects.requireNonNull(version);
+			this.withDropboxStyle = false;
+			addStep(createStep());
+		}
+
+		KtfmtConfig(String version, Boolean withDropBoxStyle) {
```

Comment:
This constructor should not exist, see my comment below in `integration_ktfmt_with_dropbox_style`.